### ISSUE TITLE
Issue loading plugin on v2.2.0

### ISSUE THREAD

**Issue by wtsnz** [wtsnz/obs-ios-camera-source#10]
I've just realized that on osx the plugin isn't loaded correctly.

```
10:11:32.909: os_dlopen(/Library/Application Support/obs-studio/plugins/obs-ios-camera-source/bin/obs-ios-camera-source.so->/Library/Application Support/obs-studio/plugins/obs-ios-camera-source/bin/obs-ios-camera-source.so): dlopen(/Library/Application Support/obs-studio/plugins/obs-ios-camera-source/bin/obs-ios-camera-source.so, 1): Library not loaded: /tmp/obsdeps/bin/libavcodec.57.dylib
10:11:32.909:   Referenced from: /Library/Application Support/obs-studio/plugins/obs-ios-camera-source/bin/obs-ios-camera-source.so
10:11:32.909:   Reason: image not found
10:11:32.909: 
10:11:32.910: Module '/Library/Application Support/obs-studio/plugins/obs-ios-camera-source/bin/obs-ios-camera-source.so' not loaded
```

Looks like the post install process didn't run correctly to patch this out.

Working on this right now..

**Comment by wtsnz** [comment]
This was caused by changes committed in af1736129f162f60f8629c8fa2b2a5b43be59669 on the `CI/install-dependencies-macos.sh` and wasn't caught in the alpha - as I had the dependencies in the tmp folder.

**Comment by wtsnz** [comment]
Fixed with v2.2.1 👍
